src/js/media-chrome-button.js

```diff
@@ -78,7 +79,7 @@ template.innerHTML = `
   }
 </style>
 
-<button></button>
+<button part="inner-button"></button>
```

Comment:
It's worth debating/discussing whether we should use a part or vars here. If we use a part, it makes it really flexible but also a breaking change if we switch to not using a native button, whereas vars could be made to work the same way.

Reply:
Do you mean if we replace `<button/>` with nothing or we replace `<button/>` with e.g. a different element? I could see how the latter would lead to a breaking change, but not the former. Regardless, wouldn't these already be true, given that each element instance has a public `nativeEl` property?